cloudify_cli/commands/profiles.py

```diff
@@ -189,13 +190,20 @@ def use(manager_ip,
             skip_credentials_validation=skip_credentials_validation,
             logger=logger,
             **kwargs)
-    cluster = env.profile.cluster.get(CloudifyNodeType.MANAGER)
-    if cluster:
-        client = env.get_rest_client(
-            client_profile=env.profile,
-            username=(env.profile.manager_username or 'admin'),
-            password=(env.profile.manager_password or 'admin')
-        )
+
+    active_profile = _get_profile(env.get_active_profile())
+    active_profile_context = env.get_profile_context(env.get_active_profile())
```

Comment:
you can run this once 'env.get_active_profile()'

Reply:
Just using `get_profile_context` should work. If no profile name is transferred to this function, it will return the current profile context. This comment relates to the next one.